server/profileFormSubmission.js

```diff
@@ -0,0 +1,35 @@
+const { Pool } = require('pg');
+const { allowedNodeEnvironmentFlags } = require('process');
+const pool = new Pool({ 
+    user: process.env.user,
+    host: process.env.host,
+    database: process.env.database,
+    password: process.env.password,
+    port: process.env.port,
+});
+
+
+module.exports.profileInsertion = async (event, callback )=> {
```

Comment:
Could we change either this or the file name?

Otherwise it becomes `profileFormSubmission.profileInsertion`. Maybe something like createCleaner.insertForm or something like that.

Reply:
".saveProfile" instead of profileInsertion is another suggestion! Makes more intuitive sense to me